ts/webui/src/components/nav/ErrorMessage.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useCallback } from 'react';
+import { Stack } from '@fluentui/react/lib/Stack';
+import { DefaultButton } from '@fluentui/react/lib/Button';
+import LogPanel from '@components/nav/slideNav/LogPanel';
+import { Important, Cancel } from '@components/fluent/Icon';
+import { EXPERIMENT } from '@static/datamodel';
+import '@style/nav/slideNavBtns.scss';
+
+export const ErrorMessage: React.FunctionComponent = () => {
+    const [hideDialog, setHideDialog] = useState(EXPERIMENT.status === 'ERROR' ? true : false);
+    const [isShowLogPanel, setShowLogPanel] = useState(false);
+    const closeLogPanel = useCallback(() => setShowLogPanel(false), []);
+    const ShowLogPanel = useCallback(() => setShowLogPanel(true), []);
+
+    return (
+        <>
```

Comment:
What's this?

Reply:
`<React.Fragment>` == `<>` == vue's `<template>`
could return many elements and do not create dom element